### DISCUSSION TITLE
typo in redeem page

### DISCUSSION THREAD

**Discussion by mucar** [IBAX-io/go-ibax#20] [Typos （English）]
Hi,

There is a typo in Redeem page: 

"Go to Werver and create a wallet address". "Werver" should be "Weaver".

my wallet address: 1168-2339-9243-7448-6536

**Comment by HalloKyle** [comment]
already received the same report again thanks for your support please left your twitter handle you will be rewarded 200 IBAX points

**Reply by mucar** [reply]
Yes, my wife was already reported it before me and she get 200 points for reporting. Thanks.
